<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoStatisticDialog</class>
 <widget class="QDialog" name="MoStatisticDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QCustomPlot" name="dailyCnt">
    <attribute name="title">
     <string>学习情况</string>
    </attribute>
   </widget>
   <widget class="QCustomPlot" name="sumCnt">
    <attribute name="title">
     <string>单词统计</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
